pricelist_per_product/README.rst

```diff
@@ -0,0 +1,59 @@
+Pricelist Per Product
+======================
+
+This module allows to see all sale prices on product form
+
+Installation
+============
+
+One click installation
```

Comment:
IMO, nothing is "mandatory". If you don't have anything useful to put in there, just don't put the section. 

I hate copy pasted docstrings which give no useful information but are there because of a policy saying "you have to put a docstring"...